src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -619,6 +621,33 @@ private String generateEvent(String descTemp, SkriptEventInfo<?> info, @Nullable
 		// Return Type
 		desc = handleIf(desc, "${if return-type}", false);
 
+		// Event Values
+		Set<String> eventValues = new TreeSet<>();
+		Multimap<Class<? extends Event>, EventValueInfo<?, ?>> allEventValues = EventValues.getPerEventEventValues();
+		for (Class<? extends Event> supportedEvent : info.events) {
+			for (Class<? extends Event> event : allEventValues.keySet()) {
+				if (!event.isAssignableFrom(supportedEvent))
+					continue;
+
+				Collection<EventValueInfo<?, ?>> eventValueInfos = allEventValues.get(event);
+
+				for (EventValueInfo<?, ?> eventValueInfo : eventValueInfos) {
+					Class<?>[] excludes = eventValueInfo.excludes();
+					if (excludes != null && Set.of(excludes).contains(event))
+						continue;
+
+					ClassInfo<?> exactClassInfo = Classes.getExactClassInfo(eventValueInfo.c());
+					if (exactClassInfo == null)
+						continue;
+
+					eventValues.add("event-%s".formatted(exactClassInfo.getName()));
+				}
+			}
```

Comment:
I'm not 100% sure this accurately shows every event-value available, but I think it's guaranteed to show all the reliable event-values. You can see EventValues#getEventValueConverters for more details